test/e2e/framework/volume_util.go

```diff
@@ -61,6 +61,8 @@ const (
 	GlusterfsServerImage string = "gcr.io/google_containers/volume-gluster:0.2"
 	CephServerImage      string = "gcr.io/google_containers/volume-ceph:0.1"
 	RbdServerImage       string = "gcr.io/google_containers/volume-rbd:0.1"
```

Comment:
Should the tags above be updated to the new versions?

Reply:
No, this PR must be merged first, then someone (from google?) must manually build & push new images and then we can bump versions here. Otherwise we blow all e2e tests.

Reply:
I agree with @jsafrane that we need to push image first before modifying testcode. @ixdy can you please help us pushing above images so that we can pass these tests with new images?

Reply:
sure, I can help build images, though currently the test-image-build-lock is assigned to #52744.

Reply:
@ixdy : As #52744 is merged, can you please help us pushing these images?

Reply:
ping @ixdy